SoundSwitch/Framework/DeviceCyclerManager/DeviceCycler/ADeviceCycler.cs

```diff
@@ -12,23 +12,19 @@
 * GNU General Public License for more details.
 ********************************************************************/
 
-using System;
 using System.Collections.Generic;
 using System.Linq;
 using NAudio.CoreAudioApi;
 using Serilog;
 using SoundSwitch.Audio.Manager;
-using SoundSwitch.Audio.Manager.Interop;
 using SoundSwitch.Audio.Manager.Interop.Enum;
-using SoundSwitch.Framework.Audio.Device;
+using SoundSwitch.Common.Framework.Audio.Device;
 using SoundSwitch.Model;
 
 namespace SoundSwitch.Framework.DeviceCyclerManager.DeviceCycler
 {
     public abstract class ADeviceCycler : IDeviceCycler
```

Comment:
_**A**DeviceCycler_? what's with B to Z 🙃 Do you mean a _SingleDeviceCycler_?

Reply:
I -> Interface
A -> Abstract :)